pkg/operator/status.go

```diff
@@ -44,6 +45,19 @@ func updateCondition(cr *imageregistryv1.Config, condtype string, condstate oper
 	}
 
 	cr.Status.Conditions = conditions
+	updateDegradedMetric(condtype, condstate)
```

Comment:
Isn't `updateCondition` called multiple times? Can't this result in a lot of increments per reconciliation loop?

Reply:
My bad, `updateCondition` is called multiple times but only once using `Degraded` condition as parameter. This never increases more than once per loop.